index.html

```diff
@@ -0,0 +1,341 @@
+				ago.
+				<span class="main-pic-caption-name">Rick Friedman for The New York Times</span>
+			</p>
+		</div>
+
+		<div class="article-segment-one">
+
+			<p class="writer">By Dennis Overbye</p>
+
+			<div class="article-header">
+				<p>March 17,2014</p>
+			</div>
+
+			<p>CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth, with a new son
+				and a year’s appointment at Stanford, stayed up late with his notebook and equations, venturing far
+				beyond the world of known physics.</p>
+
+			<p>He was trying to understand why there was no trace of some exotic particles that should have been created
+				in the Big Bang. Instead he discovered what might have made the universe bang to begin with. A potential
+				hitch in the presumed course of cosmic evolution could have infused space itself with a special energy
+				that exerted a repulsive force, causing the universe to swell faster than the speed of light for a
+				prodigiously violent instant.</p>
+
+			<p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole
+				and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and
+				irregularities. Those particles were not missing, but would be diluted beyond detection, like spit in
+				the ocean.</p>
+
+			<p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+
+			<p>On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of
+				the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.</p>
+
+			<p>Reaching back across 13.8 billion years <a
+					href="https://www.youtube.com/watch?v=ZJYc9YmKIO8&amp;feature=youtu.be">to the first sliver of
+					cosmic time</a> with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the
+				Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called <a
+					href="http://www.nytimes.com/2016/02/12/science/ligo-gravitational-waves-black-holes-einstein.html?module=inline">gravitational
+					waves</a> — the signature of a universe being wrenched violently apart when it was roughly a
+				trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun
+				evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
+
+			<p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered
+				whether it could ever be proved.</p>
+
+			<p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery
+				of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time
+				and space and energy to science and speculation.</p>
+
+		</div>
```

Comment:
- [ ]  Consider rephrasing the comments as suggestions instead of orders. For example, instead of "Use aside tag", you can give a polite suggestion as "I think it would be better to use aside tags here".
- [ ] Consider adding an explanation for each comment.  For example, explain why it is better to use an aside tag. In this way, your suggestions are valuable feedback.
-[ ] Consider giving more details to your comments. For example, you commented, "In this editor's pick section, it does not display properly". Why don't you add how should it be displayed?